pkg/firewall/firewall.go

```diff
@@ -28,10 +28,10 @@ const (
 	// We use the cache to minimize calls to Ethereum client.
 	authorizationCachePeriod = 24 * time.Hour
 
-	// activeKeepCachePeriod is the time the cache maintains information
+	// activeKeepMemberCachePeriod is the time the cache maintains information
 	// about active and no active keep members. We use the cache to minimize
 	// calls to Ethereum client.
-	activeKeepCachePeriod = 168 * time.Hour // one week
+	activeKeepMemberCachePeriod = 168 * time.Hour // one week
```

Comment:
Doesn't the use of this for `keepInfoCache` mean the original name was correct?

Reply:
It's also not clear to me that we should be using the same lifetime for active and inactive caching.

Reply:
Yes... After looking at it again I think they both are correct but the original one was shorter so probably better, heh.

Reply:
> It's also not clear to me that we should be using the same lifetime for active and inactive caching.

They can have different values. I think we historically decided for one because it fits well in both cases. 

It is quite confusing but "no active keep member" and "inactive keep" caching are two different things.
"inactive keep" is cached forever in `keepInfoCache` and means the keep with the given ID is inactive.
"no active keep member" is cached for one week and it means that the given operator is not a member in any active keep.

It could happen that someone kept trying to connect all the time to our node without a minimum stake (with a modified client) and one week cache lifetime for that address would let us block those attempts early. With a shorter caching lifetime, we'd have to execute the loop of checks more often.

In the case of a node who is a member in at least one active keep, one week cache lifetime felt like a good balance between too frequent checks and keeping this information relatively up to date.